src/flows/context/query.tsx

```diff
@@ -191,12 +209,72 @@ export const QueryProvider: FC = ({children}) => {
       })
   }
 
+  const forceUpdate = (id, data) => {
+    try {
+      update(id, data)
+    } catch (_e) {
+      add(id, data)
+    }
+  }
+
+  const queryAll = () => {
+    if (isLoading === RemoteDataState.Loading) {
+      return
+    }
+
+    const map = generateMap(runMode === RunMode.Run)
+
+    if (!map.length) {
+      return
+    }
+
+    event('Flow Submit Button Clicked')
```

Comment:
this event should exist on the submit button only, and not on the underlying implementation of the submit logic. then if we wanna change how we submit something, the events in GA keep ticking

Reply:
Totally